livy/src/main/java/org/apache/zeppelin/livy/LivySparkSQLInterpreter.java

```diff
@@ -80,7 +80,8 @@ public InterpreterResult interpret(String line, InterpreterContext interpreterCo
               line.replaceAll("\"", "\\\\\"")
                   .replaceAll("\\n", " ")
               + "\").show(" +
-              property.get("zeppelin.livy.spark.sql.maxResult") + ")",
+              property.get("zeppelin.livy.spark.sql.maxResult") + "," +
+              property.get("zeppelin.livy.spark.sql.truncateResult") + ")",
           interpreterContext, userSessionMap);
```

Comment:
There will be an exception thrown. I don't think every property must be checked for validity in an explicit way.